Dockerfile_proxy

```diff
@@ -0,0 +1,10 @@
+FROM python:2.7
+ENV PYTHONUNBUFFERED 1
+RUN mkdir /code
+WORKDIR /code
+ADD requirements.txt /code/
```

Comment:
This is redundant as we can run `pip install -r requirements.txt` after the `ADD` below.